### ISSUE TITLE
export favorites: directory items are displayed randomly

### ISSUE THREAD

**Issue by Vrihub** [segler-alex/RadioDroid#796]
When you export favorites, you are presented with a directory listing that displays files and subdirectories in random order; I would expect them sorted alphabetically.

**Comment by Vrihub** [comment]
I forgot: this also happens when saving history.

Version 0.83 on Android 9.

**Comment by morckx** [comment]
Yes, it would be easy to fix, but the file selector library used by RadioDroid has been abandoned for a long time and it makes no sense that we repair it ourselves. 

The best thing would be to introduce a case distinction and use the standard storage access framework (ACTION_GET_CONTENT, ...) from API19 (?) on. This should be easy and would also resole #761 and #480
